sdk/identity/Azure.Identity/README.md

```diff
@@ -109,92 +86,137 @@ using Azure.Security.KeyVault.Secrets;
+var credential = new ChainedTokenCredential(new ManagedIdentityCredential(), new AzureCliredential());
 
-var keyClient = new KeyClient(new Uri("https://myvault.azure.vaults.net/"), credential);
+var eventHubProducerClient = new EventHubProducerClient("myeventhub.eventhubs.windows.net", "myhubpath", credential);
 ```
-### Authenticating a user with the default browser
-The `InteractiveBrowserCredential` allows an application to authenticate a user by launching the system's default browser. This example demonstrates authenticating the `BlobClient` from the [Azure.Storage.Blobs][blobs_client_library] client library using the `InteractiveBrowserCredential`.
 
-```c#
-using Azure.Identity;
-using Azure.Storage.Blobs;
+## Credential Classes
+### Authenticating Azure Hosted Applications
+|credential  | usage
+|-|-|-
+|`DefaultAzureCredential`|provides a simplified authentication experience to quickly start developing applications run in the Azure cloud
+|`ChainedTokenCredential`|allows users to define custom authentication flows composing multiple credentials
+|`EnvironmentCredential`|authenticates a service principal or user via credential information specified in environment variables
+|`ManagedIdentityCredential`|authenticates the managed identity of an azure resource
 
-// authenticating a service principal with InteractiveBrowserCredential
-var credential = new InteractiveBrowserCredential(clientId);
+### Authenticating Service Principals
+|credential  | usage
+|-|-|-
+|`ClientSecretCredential`|authenticates a service principal using a secret
+|`CertificateCredential`|authenticates a service principal using a certificate
 
-var blobClient = new BlobClient(new Uri("https://myaccount.blob.core.windows.net/mycontainer/myblob"), credential);
-```
-__Note:__ If a default browser is not available in the system, or the current application does not have permissions to create a process authentication with the `InteractiveBrowserCredential` will fail with an `AuthenticationFailedException`.
-### Authenticating a user with the device code flow
 
-The device code authentication flow allows an application to display a device code to a user, and then the user will authenticate using this code through a browser, typically on another client.  This authentication flow is most often used on clients that have limited UI and no available browser, such as terminal clients and certain IoT devices.  
+### Authenticating Users
+|credential  | usage
+|-|-|-
+|`InteractiveBrowserCredential`|interactively authenticates a user with the default system browser
+|`DeviceCodeCredential`|interactively authenticates a user on devices with limited UI
+|`UserPasswordCredential`|authenticates a user with a username and password
+|`AuthorizationCodeCredential`|authenticate a user with a previously obtained authorization code
+
+### Authenticating via Development Tools
+|credential  | usage
+|-|-|-
+|`AzureCliCredential`|authenticate in a development environment with the Azure CLI
```

Comment:
Do you want these to link reference docs?

Reply:
This is a good suggestion I'm going to do a follow up PR which fixes all the relative links included in this PR and I'll update these at the same time.